plugins/fileservices/fileservices.cpp

```diff
@@ -142,6 +142,9 @@ static const char * EclDefinition =
 "  varstring GetEspURL(const varstring username = '', const varstring userPW = '') : c,once,entrypoint='fsGetEspURL'; \n"
 "  varstring GetDefaultDropZone() : c,once,entrypoint='fsGetDefaultDropZone'; \n"
 "  dataset(FsDropZoneRecord) GetDropZones() : c,context,entrypoint='fsGetDropZones'; \n"
+"  integer4 GetExpireDays(const varstring lfn) : c,context,entrypoint='fsGetExpireDays'; \n"
+"  SetExpireDays(const varstring lfn, integer4 expireDays = -1) : c,context,entrypoint='fsSetExpireDays'; \n"
```

Comment:
Should remove default, which doesn't make sense now since SetExpireDays doesn't except -1

Reply:
Removed